### ISSUE TITLE
Sitewide Password Protection Bug

### ISSUE THREAD

**Issue by marywarrington** [statamic/v2-hub#1001]
### Expected behaviour

I implemented sitewide protection, but have to enter the password every time I navigate to a new page on the site.
### Actual behaviour

I'd like it to remember that I've already entered the password, and allow me to navigate freely for the session!
### Steps to reproduce
1.  In system.yaml

```
protect:
  password:
    allowed: ['testpassword']
    form_url: /password-entry
```
1.   In routes.yaml

```
routes:
  /password-entry:
    template: password-entry
    protect: false
```
1.  Make a new template (password-entry.html)

```
            {{ protect:password_form }}
                {{ if no_token }}
                    <p>No token has been provided.</p>
                {{ else }}
                    {{ if error }}
                        <p class="error">{{ error }}</p>
                    {{ /if }}

                    <h3>Please enter your password:</h3>
                    <input type="password" name="password" />

                    <button type="submit" class="button">Submit</button>
                {{ /if }}
            {{ /protect:password_form }}
```
1. I also pasted new code (provided by jason) into my statamic > bundles > protect > protectors > PasswordProtector.php.
   ### Server configuration
   **Operating system**: OSX

**Web server:** currently local. MAMP.

**PHP version:** 7.0.8

**Statamic version:** 2.1.11

**Updated from an older Statamic or fresh install:** Updated

**Comment by mikemartin** [comment]
Experiencing the same issue on a staging server. Sounds like it might be a known issue.
https://lodge.statamic.com/questions/2535-protecting-content-allow-logged-in#reply-1

**Comment by jasonvarga** [comment]
This is fixed in 2.6 - with a slightly different syntax. If you still need to use this feature, check out the docs for it when it's out.

**Comment by sicbe** [comment]
I'm having the exact same "problem" with folder protection. I wanted to protect a folder with a password, but now I have to enter the password again for every page in the folder, while it's the same password for the entire folder. Is this the same bug, or is it intented to work that way?
